### PR TITLE
Add additional columns to IPAddress model admin

### PR DESCRIPTION
**Pull request by szok**
Added:
- Additional columns to IPAddress
- Raw ID Fields to IPAddress and Rack

### CODE REVIEW

File: src/ralph/settings/dev.py [hunk 1]
```diff
@@ -21,3 +21,5 @@ def only_true(request):
 }
 
 ROOT_URLCONF = 'ralph.urls.dev'
+
+RAISE_ITEMS_ERRORS_ON_DEBUG = not DEBUG
```
Comment (vi4m) [inline src/ralph/settings/dev.py]: what is this? If it's really needed, comment it clearly

File: src/ralph/settings/dev.py [hunk 2]
```diff
@@ -21,3 +21,7 @@ def only_true(request):
 }
 
 ROOT_URLCONF = 'ralph.urls.dev'
+
+# Disable exception for the missing element in SiteTree
+# https://github.com/idlesign/django-sitetree/pull/157/files
+RAISE_ITEMS_ERRORS_ON_DEBUG = not DEBUG
```
Comment (mkurek) [inline src/ralph/settings/dev.py]: why not explicitly False?

**Comment by mkurek** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Pull request merged**